data/alt-image-text.md

```diff
@@ -1,6 +1,6 @@
 ### Images with text should be accessible to screen readers
 
-Images that contain text in them such as logos should make that text available to screen readers in some form. The most straightforward way to do this is to style a `header` element as image and use `text-indent: -9999px` as its text content.
+Images that contain text in them such as logos should make that text available to screen readers in some form. It's best to place the image using the CSS `background-image` attribute with the text content in the HTML. There are a [variety of methods](https://css-tricks.com/the-image-replacement-museum/) to hide the text, but typically `color: transparent`, `overflow: hidden`, and `user-select: none` is effective.
```

Comment:
I think that the link should be a resource and not linked via the snippet's text itself.